test/integration/version-18.03.1/testinfra/test_docker.py

```diff
@@ -2,9 +2,9 @@
 
 
 def test_package_is_installed(Package):
-    docker = Package('docker-engine')
+    docker = Package('docker-ce')
     assert docker.is_installed
-    assert docker.version.startswith('1.12.2')
+    assert docker.version.startswith('18.03.1')
```

Comment:
Could the package version be configurable?

Reply:
See #153 could you use `docker:pkg:version` here (and elsewhere) please?

Reply:
Can't use a saltstack pillar var in test python code without non-trivial work

Reply:
Hi @javierbertoli @blbradley I'm okay with not complicating things so leave this as is!! thx